spring/boot-autoconfigure/src/test/java/com/linecorp/armeria/spring/ArmeriaAutoConfigurationTest.java

```diff
@@ -91,14 +98,20 @@
     public static class TestConfiguration {
         @Bean
         public AnnotatedServiceRegistrationBean annotatedService() {
+            final AnnotatedService service = new AnnotatedService();
             return new AnnotatedServiceRegistrationBean()
                     .setServiceName("annotatedService")
-                    .setService(new AnnotatedService())
+                    .setService(service)
                     .setPathPrefix("/annotated")
                     .setDecorators(LoggingService.newDecorator())
                     .setExceptionHandlers(ImmutableList.of(new IllegalArgumentExceptionHandler()))
                     .setRequestConverters(ImmutableList.of(new StringRequestConverterFunction()))
-                    .setResponseConverters(ImmutableList.of(new StringResponseConverter()));
+                    .setResponseConverters(ImmutableList.of(new StringResponseConverter()))
+                    .setExampleRequests(Collections.singleton(ExampleRequest.of(service.getClass().getName(),
```

Comment:
nit: How about changing `Collections.singleton(...)` -> `ImmutableList.of(...)` to keep consistency?

Reply:
Thank you for review. I fixed it in 5dd296c.